docs/en/seatunnel-engine/rest-api.md

```diff
@@ -262,3 +262,91 @@ network:
 
 </details>
 
+------------------------------------------------------------------------------------------
+
+### Encrypt Config.
+
+<details>
+<summary><code>POST</code> <code><b>/hazelcast/rest/maps/encrypt-config</b></code> <code>(Returns the encrypted config if config is encrypted successfully.)</code></summary>
+For more information about customize encryption, please refer to the documentation [config-encryption-decryption](../connector-v2/Config-Encryption-Decryption.md).
+#### Body
```

Comment:
```suggestion

#### Body

```
![image](https://github.com/apache/seatunnel/assets/32387433/119eea0e-2ae9-4200-9286-d5825e2b00b3)
The format have some problem.

Reply:
done.